builders/promote.yaml

```diff
@@ -0,0 +1,106 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Promotes the revision of a service to receive 100% of traffic.
+#
+
+substitutions:
+  _REGION:
+  _REVISION:
+
+steps:
+- id: export
+  name: gcr.io/google.com/cloudsdktool/cloud-sdk:293.0.0-alpine
+  args:
+  - 'bash'
+  - '-c'
+  - |-
+    gcloud run services update-traffic export \
+      --quiet \
+      --project "${PROJECT_ID}" \
+      --platform "managed" \
+      --region "${_REGION}" \
+      --to-revisions "${_REVISION}=100"
```

Comment:
where are we getting the revision name from?

Reply:
`./scripts/promote`. Default is "LATEST", but can override.